gui/src/main/proxy.js

```diff
@@ -0,0 +1,107 @@
+/* eslint standard/no-callback-literal: 0 */
+
+const ElectronProxyAgent = require('electron-proxy-agent')
+const url = require('url')
+const http = require('http')
+const https = require('https')
+
+const log = require('cozy-desktop').default.logger({
+  component: 'GUI:proxy'
+})
+
+const config = require('yargs')
```

Comment:
Why not reuse `commander` since it was already used in CLI? Is it obsolete in some way? But same a CLI-related comment above: not a priority (just out of curiosity).

Reply:
It was in the CLI package and I am scared of the dark voodoo between cli & gui if they use the same package 😃